docs/installing-northstar/basic-setup.md

```diff
@@ -4,22 +4,25 @@
 
 {% embed url="https://www.youtube.com/watch?v=bK4pV-AHOHM" %}
 
-{% hint style="info" %}
-TODO: Method to get to Origin's local files described below is incorrect/doesn't exist
-{% endhint %}
-
-Firstly note that Northstar client is only available on PC and requires you to **both own the game and have it installed**.
+Firstly, note that the Northstar client is only available on PC and requires you to **both own the game and have it installed**.
 
 1. Download the latest version of Northstar from the [releases](https://github.com/R2Northstar/Northstar/releases) page
-2. Copy all the files in the newly downloaded zip folder to your Titanfall folder
-   * **For Steam** - Right click _Titanfall 2_ > Open _Properties_ > Click _Local Files_ > Click _Browse_
-   * **For Origin** - Click Titanfall 2 in the Library > Click the gear icon > Click _Game Properties_
+2. Copy all the files in the newly downloaded zip folder to your Titanfall2 folder. This can vary depending on whether you purchased the game off Steam or Origin, and if you set a custom folder for game installs.
+   ```
+   examples:
+      C:\Program Files (x86)\Origin Games\Titanfall2
+      C:\Program Files (x86)\Steam\steamapps\common\Titanfall2
+      D:\MyCustomGameFolder\Titanfall2
+   ```
+   {% hint style="info" %}
+   For those on Steam, you don't have to go folder hunting! In your Steam library, right click _Titanfall 2_ > Open _Properties_ > Click _Local Files_ > Click _Browse_
+   {% endhint %}
```

Comment:
Maybe something like this?

```suggestion
   * **For Steam** - Right click _Titanfall 2_ > Open _Properties_ > Click _Local Files_ > Click _Browse_\
   Usually: `C:\Program Files (x86)\Steam\steamapps\common\Titanfall2`
   * **For Origin** - To find the location of your Origin library go to _Origin_ -> _Application Settings_ -> _Installs & Saves_ -> _On your Computer_ -> _Game Library Location_\
   Go to the directory mentioned there using File Explorer and open the `Titanfall2` folder.\
   Usually: `C:\Program Files (x86)\Origin Games\Titanfall2`
```

Reply:
I think that works. It also serves as a much clearer explanation of what to do.